### PR TITLE
Misp v3 delete attribute + local tag arg

### PR DESCRIPTION
**Pull request by abaumgarten**
## Status
- [] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/46774

## Description
- Added new command ***misp-delete-attribute***.
- In the **misp-add-tag-to-event** command the *is_local* argument was added.
- In the **misp-add-tag-to-attribute** command the *is_local* argument was added.

## Screenshots
*1
image
*2
image
*3
image

## Minimum version of Cortex XSOAR
- [x] 5.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [x] Tests - TBP was updated.
- [x] Documentation  - readme was updated.

@bakatzir FYI

### CODE REVIEW

File: Packs/MISP/pack_metadata.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
     "name": "MISP",
     "description": "Malware information and threat sharing platform.",
     "support": "xsoar",
-    "currentVersion": "2.0.6",
+    "currentVersion": "2.0.7",
```
Comment (bakatzir) [inline Packs/MISP/pack_metadata.json:5]: Should bump a minor as I see it https://semver.org/.

File: Packs/MISP/ReleaseNotes/2_0_7.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+#### Integrations
+##### MISP v3
+- Added new command ***misp-delete-attribute***.
```
Comment (bakatzir) [inline Packs/MISP/ReleaseNotes/2_0_7.md:3]:
```suggestion
- Added the command ***misp-delete-attribute*** which allow users to delete an attribute according to the given attribute ID.
```

File: Packs/MISP/ReleaseNotes/2_1_0.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
+#### Integrations
+##### MISP v3
+- Added the command ***misp-delete-attribute*** which allow users to delete an attribute according to the given attribute ID.
+- In the **misp-add-tag-to-event** command the *is_local* argument was added.
+- In the **misp-add-tag-to-attribute** command the *is_local* argument was added.
```
Comment (guykeller) [inline Packs/MISP/ReleaseNotes/2_1_0.md:5]:
```suggestion
- Added the command ***misp-delete-attribute*** which allows users to delete an attribute according to a given attribute ID.
- Added the *is_local* argument to the **misp-add-tag-to-event** command.
- Added the *is_local* argument to the **misp-add-tag-to-attribute** command.
```

**Review by guykeller — APPROVED** [review]
Please just fix the documentation (per my note), and consider if UTs should be added even though the TPB was enhanced

**Pull request merged**